components/game/game.tsx

```diff
@@ -36,6 +36,9 @@ const GameLayout: NextPage = () => {
   };
 
   const handleNextTrack = () => {
+    if (gameState.currentSong + 1 === gameState.playlist.length) {
+      router.push('/summary');
+    }
     dispatch(gameSlicesActions.nextGame());
```

Comment:
Is not neccesary to dispatch `nextGame`at the end.

Reply:
Ready Sir